pages/memgraph-lab/querying.mdx

```diff
@@ -0,0 +1,313 @@
+node from the canvas. Besides clicking on a node and then on the button, you can
+also expand and collapse nodes with a double-click.
+
+![](/pages/data-visualization/lab-user-manual/graph-results.png)
+
+{<h4 className="custom-header">Graph customization</h4>}
+
+To customize the appearance of the graph you can update the GSS in the [Graph
+Style editor](#graph-style-editor) or change the **Display options** in the
+property sidebar which you can access by clicking on a graph object you'd like
+to customize. 
+
+Display options offers you the ability to change the color or text on display
+for the selected graph object or all the graph objects with the same label or of
+the same type.
+
+In the bottom left corner, you can **Enable physics**, that is, make nodes
+interact with each other, by pulling away or closing in to one another,
+depending on the strength of the relationships between them.
+
+In the top right corner of the graph you can open **Graph Preferences** and set
+the collision radius that dictates the margin radius for each node from its
+center, the repel force that dictates how strongly nodes repel each other, and
+the link distance that dictates the minimum required distance between two
+connected nodes.
+
+![](/pages/data-visualization/lab-user-manual/physics.png)
+
+Another interesting feature you can use on graph data results is the map
+background. This feature automatically turns on when the result nodes have
+numeral `lat` and `lng` properties.
+
+![](/pages/data-visualization/lab-user-manual/map.png)
+
+If latitude and longitude data are not stored in the default `lat` and `lng`
+properties, you can use a Graph Style Script (GSS) to map alternative fields for
+location tracking. For example:
+
+```cpp
+// nodes have .latitude and .longitude fields
+Define(LATITUDE_FIELD, "latitude")
+Define(LONGITUDE_FIELD, "longitude")
+
+// In the case of numeric latitude and longitude properties, set them up for a switch to a map view
+@NodeStyle And(IsNumber(Property(node, LATITUDE_FIELD)), IsNumber(Property(node, LONGITUDE_FIELD))) {
+  latitude: Property(node, LATITUDE_FIELD)
+  longitude: Property(node, LONGITUDE_FIELD)
+}
+```
+
+
+## Query summary
+
+After executing a query in Memgraph Lab, the query summary provides essential
+metadata and insights into the execution process. This summary helps users
+evaluate the performance, efficiency, and impact of their queries.
+
+![](/pages/data-visualization/querying/query-summary.png) 
+
+The query summary shows important information involving:
+- **Database impact statistics**: Displays key database impact metrics such -
+  the number of hops traversed, the number of nodes and relationships created or
+  delete, the number of labels added or removed, the number of indexes and
+  constraints created or deleted, the number of properties set 
+- **Query performance breakdown (ms)**: For complex queries, Memgraph Lab
```

Comment:
Lab 3.2:

Removed query stats on indexes and constraints
  in the summary, as they aren't provided by
  Memgraph.